src/journey-maps/angular/components/attribution/attribution.html

```diff
@@ -6,9 +6,9 @@
   aria-hidden="true"
 >
   <div class="attribution-text">
-    <span *ngFor="let attribution of attributions">
-      <span [innerHTML]="attribution" class="attribution-item"></span>
-    </span>
+    <ng-container *ngFor="let attribution of attributions">
```

Comment:
is the ng-container really needed or could "*ngFor="let attribution of attributions"" be applied to the div? I am not sure, because of the `innerHTML` if it really works :)

Reply:
I tried your suggestion and it works:

```
    <div *ngFor="let attribution of attributions" [innerHTML]="attribution" class="attribution-item"></div>
```

But in the end I decided not to push the change as I find it more readable the way it was before.